main-project/src/chats/chats-controller.service.ts

```diff
@@ -218,23 +220,21 @@ export class ChatsControllerService {
       userNo: inviterNo,
       chatRoomNo,
       isUserNeeded: true,
-      target: `초대하신 ${targetUserNo}`,
     });
 
     await this.checkUserInChatRoom({
       userNo: targetUserNo,
       chatRoomNo,
       isUserNeeded: false,
-      target: `${targetUserNo}`,
     });
 
     await this.joinChatRoom({ userNo: targetUserNo, chatRoomNo, userType });
   }
 
-  private async joinChatRoom(chatUserInfo: ChatUserInfo): Promise<void> {
+  private async joinChatRoom(chatUserInfo: ChatUser): Promise<void> {
     const user = [chatUserInfo];
 
-    const affectedRow = await this.chatUsersRepository.setChatRoomUsers(user);
+    const affectedRow = await this.chatUsersRepository.createChatUsers(user);
```

Comment:
```suggestion
    const affectedRow: number = await this.chatUsersRepository.createChatUsers(user);
```

Reply:
repo에서 반환타입을 ResultSetHeader로 하면
```suggestion
    const { affectedRows } : ResultSetHeader = await this.chatUsersRepository.createChatUsers(user);
```
도 가능합니다